stdlib/public/Platform/winsdk.modulemap

```diff
@@ -217,6 +217,11 @@ module WinSDK [system] {
     link "WinMM.Lib"
   }
 
+  module Misc {
+    header "Windows.h"
```

Comment:
This really is incorrect.  `Windows.h` is an umbrella header, not a miscellaneous module.

Reply:
Uh, I don't think it's possible to make it an umbrella header without changes to SourceKit or Clang, so I'll just drop this commit.